Demo/Program.cs

```diff
@@ -7,14 +7,19 @@ class Program
     {
         static void Main(string[] args)
         {
+            /*
             Document doc = new();
```

Comment:
Let's move this commented sample code into a new method called "TestTextWriter". Let's change the string "text field I like you from Poland" to just "Hello World!".  Let's just use an inbuilt font, not "kenpixel.ttf" e.g.:
`writer.FillTextbox(page.Rect, "Hello World!", new Font(fontName: "helv"), rte: true);`